### ISSUE TITLE
s.charCodeAt is not a function

### ISSUE THREAD

**Issue by Chlwjdgh1990** [Aymkdn/ExcelPlus#22]
Hi,

I followed your first example as shown on your homepage. 

`var ep = new ExcelPlus();`
`ep.createFile("Book1")`
`.write({"content": [["A1", "B1", "C1"]]})`
`.createSheet("Book2")`
                `.write({ "cell": "A1", "content": "A1" })`
               ` .write({ "sheet": "Book1", "cell": "D1", "content": new Date() })`
                `.saveAs("demo.xlsx");`

But I keep getting:

Uncaught TypeError: s.charCodeAt is not a function
    at s2ab (excelplus-2.5.js:658)
    at ExcelPlus.saveAs (excelplus-2.5.js:661)

What could be the issue here?

**Comment by Aymkdn** [comment]
Which browser do you use?

**Comment by Chlwjdgh1990** [comment]
I'm currently on Chrome

**Comment by Aymkdn** [comment]
I've just tried [http://aymkdn.github.io/ExcelPlus/demo/example1.html](http://aymkdn.github.io/ExcelPlus/demo/example1.html) in Chrome, and it doesn't return an error.... Does this example page work for you?

**Comment by Chlwjdgh1990** [comment]
Yes, it does. This is weird. Why does it not work for me? I did exactly the same.

But when I print wbout in line 654, I get this:

Promise {[[PromiseStatus]]: "pending", [[PromiseValue]]: undefined}

Update: I forgot to mention that I had to modify xlsx.js: . z.generate() in `function write_zip_type(wb, opts)` had to be renamed to z.generateAsync() because generate() was deprecated and thus no longer available in  JSZip. Any ideas how to get this working with newer Versions of JSZip?

Update2: I got now! I had to use an older version of JSZip because JSZip API changed and they completely removed the sync interface.
